swagger-ui/src/main/resources/index.html

```diff
@@ -1,142 +0,0 @@
-<!--
```

Comment:
What is the downside of keeping the swagger UI? The most important thing is the swagger.json as @jimmidyson said, but if the cost of keeping the swagger UI is exactly the same and maintenance is not an issue (just slows down deployment a bit), I don't see why keeping it is a big deal.